src/core_plugins/metrics/server/lib/get_vis_data.js

```diff
@@ -0,0 +1,60 @@
+import _ from 'lodash';
+import moment from 'moment';
+import Color from 'color';
+import calculateLabel from '../../public/components/vis_editor/lib/calculate_label';
```

Comment:
The `calculateLabel` is used on both the front-end and back-end so either there will be a large directory traversal on the front-end or on the back-end.